src/index.d.ts

```diff
@@ -54,6 +55,9 @@ declare module 'react-idle-timer' {
     'mouseout' |
     'mouseup' |
     'mousewheel' |
+    'mouseWheel' |
```

Comment:
@SupremeTechnopriest  is it intentional to have `mouseWheel` & `mousewheel`?

One already existed in the ts type, the other existed in the default types.

Reply:
No it should just be `mousewheel`.  https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event

If the default types have mouseWheel then thats a type-o and we should probably fix it.

Reply:
Thanks, I have added a commit to address the typo.

Reply:
Looks good to me! Thanks for the PR.  Merging in now and will push to NPM later tonight.